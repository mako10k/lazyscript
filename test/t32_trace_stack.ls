# 1行目で未定義参照を発生させる（Ref経由）
!{ ~undefined_symbol };
