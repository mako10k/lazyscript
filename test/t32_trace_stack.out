E: /workspace/lazyscript/test/t32_trace_stack.ls:2.4-21: undefined reference: ~undefined_symbol
E: #err "undefined reference"
